app/assets/stylesheets/application.scss

```diff
@@ -1362,3 +1362,240 @@ div.field {
+    padding-right: 0;
+  }
+
+  .main-header {
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+
+  }
+
+  .header-subnav {
+    display: inline-block;
+    width: 100%;
+    margin: 0;
+  }
+
+  .wrap-page {
+    margin-top: 25px;
+  }
+}
+
+@media screen and (max-width: 800px){
+
+  .header {
+    width: 100%;
+    height: 303px;
+    display: block;
+    padding: 0;
+    margin: 0;
+  }
+
+  header.header h1.main-header {
+    background-color: $primary-color;
+    margin: 0;
+    width: 100%;
+
+    a {
+      font-size: 2rem;
+      width: 100%;
+      padding-top: .25rem;
+      padding-bottom: .25rem;
+    }
+  }
+
+  .header-subnav {
+    margin-left: 0;
+    margin-right: 0;
+    height: 255px;
+    margin-top: -60px;
+
+    li {
+      width: 100%;
+    }
+  }
+
+  .header-subnav li {
+    a.is-active {
+      background-color: $secondary-transparent-color;
+    }
+  }
+
+  .center-content {
+    display: block;
+  }
+
+  .adventures-search form {
+    width: 526px;
+    margin: auto;
+  }
+
+  input.adventures-search-bar {
+    font-family: 'Architects Daughter', courier;
+    font-size: 1.2rem;
+    height: 45px;
+    width: 350px;
+    margin: 0;
+  }
+}
+
+@media screen and (max-width: 700px){
+
+  .header {
+    width: 100%;
+    height: 303px;
+    display: block;
+    padding: 0;
+    margin: 0;
+  }
+
+  header.header h1.main-header {
+    margin: 0;
+
+    a {
+      text-align: center;
+      font-size: 2rem;
+      padding-top: .25rem;
+      padding-right: 37%;
+      padding-bottom: .25rem;
+      padding-left: 45%;
+    }
+  }
+
+  .header-subnav {
+    margin-left: 0;
+    margin-right: 0;
+    height: 255px;
+    margin-top: -60px;
+
+    li {
+      width: 100%;
+    }
+  }
+
+  .header-subnav li {
+    a.is-active {
+      background-color: $secondary-transparent-color;
+    }
+  }
+
+  .center-content {
+    display: block;
+  }
+
+  .adventures-search form {
+    width: 426px;
+    margin: auto;
+  }
+
+  input.adventures-search-bar {
+    font-family: 'Architects Daughter', courier;
+    font-size: 1.2rem;
+    height: 45px;
+    width: 250px;
+    margin: 0;
+  }
+}
+
+
+@media screen and (max-width: 475px){
+
+  header.header ul.header-subnav li {
+    a.is-active {
+      background-color: $secondary-transparent-color;
+    }
+  }
+
+  .adventures-search {
+    display: block;
+  }
+
+  .adventures-search form {
+    display: block;
+    width: 90%;
+    height: 100px;
+    margin: auto;
+  }
+
+  .adventures-search input {
+    display: block;
+  }
+
+  input.adventures-search-bar {
+    width: 100%;
+    margin-bottom: 10px;
+  }
+
+  input.search-adventures-button {
+    width: 200px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .center-content {
+    width: 90%;
+    padding: 0;
+    margin: auto;
+  }
+
+  .grid-groups {
+    font-size: 2rem;
+  }
+
+  .display-adventure-title {
+    padding: 25px 0 0 0;
+  }
+
+  .show-bucket-list {
+    padding: 25px 0 0 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `25px 0 0` instead of `25px 0 0 0`